Food Ordering/index.html

```diff
@@ -67,25 +66,27 @@ <h1 id="our-client-section" >OUR CLIENTS</h1>
 		</div>
 	</section>
 
-	<h1 id="contact-us">CONTACT US</h1>
+	<h1>CONTACT US</h1>
 	<section id="contact">
 		<div id="box">
```

Comment:
@astha2412 @VidhiBhatt01 actually this branch is not updated that's why it is showing changes in food ordering template also 

help her in updating the branch or making  a new branch